### ISSUE TITLE
Needs to be clear at purchase that the EXE will do nothing without internet

### ISSUE THREAD

**Issue by 8086x64** [ChrisTitusTech/winutil#2945]
Ok so I was setting up another PC and remembered the Chris Titus thingy, so went to download it but it's pay. Ok, so I bought it. Well it just crashes, err, closes immediately at launch. I used my cellphone's slow-mo camera to see if there was a message, and sure enough, for about 1/240th of a second, or whatever, I see it's trying to get online and then just quitting without letting anyone know what's up. That sucks. 

I know it used to work offline, and now it's not working. I would like a refund. If you come out with a totally off-line version that just debloats and makes a PC just tight, light, and secure as possible before connecting to any kind of internet, then I will reconsider, even if your price goes up. 

I guess maybe this is what I need to push me over to Linux, which I FGGKIN' HATE. But if I can't get a secure windows PC before it's online, then I guess it doesn't matter how convenient it is if all it's good for is games.

**Comment by NotYourAverageGamer** [comment]
A few things:

1. Winutil is free to use, with the _**OPTION**_ to pay for the executable, as a way to show support to Chris.
If you want to use it for free then you can run `irm "https://christitus.com/win" | iex`, as documented in the readme of this repository, but it *will* require internet.

2. This is not intended as an offline tool (to the best of my knowledge). I believe you could clone the repo and build yourself if you wanted to use it offline, but I can't say for sure as i haven't done this myself. The other option is to create your own ISO.

3. Winutil won't "make your pc secure", its main functionality is being a debloat tool (removing windows junk many of us dont want), and to make it easier for users to install commonly installed applications (which requires internet) without opening a browser.

**Comment by 8086x64** [comment]
_OPTION to pay for the executable, as a way to show support to Chris._

It didn't feel like a donation. If that's all it is, it could be called that? There were other support options for less money that sounded like a donation. 

The EXE is doing exactly what your command line is doing. I mean, in my day, an EXE used to do a thing. Although in this age of useless stub downloaders I guess it's not uncommon. 

_This is not intended as an offline tool (to the best of my knowledge)._

Seems it used to work off the internet. I don't know exactly when it stopped working offline as I haven't been following and supporting this as closely as perhaps I should have been.  I just know this used to be working offline, and it could be made to return to doing so.

_Winutil won't "make your pc secure"_

My understanding(which may be flawed, but I don't think so) is that the existence of fewer windows components that you do not need or want, the more secure your PC is. Especially the ones they said they would reel in, and then now they're saying they'll be installed on every PC and it's mandatory.

**Comment by 8086x64** [comment]
I did not intend to close this. What a retarded site "feature"

**Comment by 8086x64** [comment]
_OPTION to pay for the executable, as a way to show support to Chris._

It didn't feel like a donation. If that's all it is, it could be called that? There were other support options for less money that sounded like a donation. 

The EXE is doing exactly what your command line is doing. I mean, in my day, an EXE used to do a thing. Although in this age of useless stub downloaders I guess it's not uncommon. 

_This is not intended as an offline tool (to the best of my knowledge)._

Seems it used to work off the internet. I don't know exactly when it stopped working offline as I haven't been following and supporting this as closely as perhaps I should have been.  I just know this used to be working offline, and it could be made to return to doing so.

_Winutil won't "make your pc secure"_

My understanding(which may be flawed, but I don't think so) is that the existence of fewer windows components that you do not need or want, the more secure your PC is. Especially the ones they said they would reel in, and then now they're saying they'll be installed on every PC and it's mandatory.

**Comment by thecatontheceiling** [comment]
I have to agree with the OP here. The EXE feels like buying a (very low effort) product and not a donation. If you wanted to make it a donation just make it a donation instead of selling a wrapper around ``irm https://christitus.com/win | iex```. 

The fact that it's not even offline isn't mentioned anywhere clearly other than the website saying it's a "wrapper".

This is nowhere near enough because this can mean two things: It's a wrapper around the actual script or it's just a useless program that runs a command in powershell.

Chris cannot use "but it's to support me" as an excuse because the button literally says "Buy Now" and not "Donate".

**Comment by zmileski** [comment]
I agree

**Comment by ChrisTitusTech** [comment]
Jayz2cents said in the latest video about Winutil that the exe was fully offline, which is incorrect. However, due to the demand, I am working on making an offline version that will exclude online options like software installs and other online only parts of the toolbox.

**Comment by ChrisTitusTech** [comment]
Offline Access was added last month.
